lua/dap.lua

```diff
@@ -1104,10 +1103,9 @@ function M.step_over()
   session:_step('next')
 end
 
-function M.step_into()
+function M.step_into(ask_for_targets)
```

Comment:
I wonder if we should make this an `opts` dictionary already to make it easier to extend it without changing the signature or forcing users to pass unused arguments.

Could imagine that we might want to add a function to pre-filter the targets (e.g. to have the "only targets that match the current line" behavior that you mention in the PR description).

But also fine for me to merge as is - let me know what you think.

Reply:
yes, options is better. It could include also the granularity and more stuff if needed later.